### PR TITLE
Add a .babelrc file

### PR DESCRIPTION
**Pull request by carbonin**
This fixes a build issue where UglifyJS was failing in production
mode.

Specifically the error was:
ERROR in manageiq-graphql/graphql-explorer-7be988ee1dd5c4de948f.js from UglifyJs
Unexpected token: punc ()) [../manageiq-graphql-55a4ec981f49/app/javascript/graphql-explorer/index.js:5,15][manageiq-graphql/graphql-explorer-7be988ee1dd5c4de948f.js:60,58]

@himdel @imtayadeway

### CODE REVIEW

**Review by imtayadeway — APPROVED** [review]

**Comment by Glutexo** [comment]
Isn’t it possible that this broke `rake update:ui`? I get a ton of errors like this after updating:

```
ERROR in /Users/stomsa/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/bundler/gems/manageiq-graphql-ce3bd9ed5bef/node_modules/setimmediate/setImmediate.js
Module build failed: Error: Couldn't find preset "es2015" relative to directory "/Users/stomsa/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/bundler/gems/manageiq-graphql-ce3bd9ed5bef"
    at /Users/stomsa/Vývoj/ManageIQ/Repozitáře/manageiq-ui-classic/node_modules/babel-core/lib/transformation/file/options/option-manager.js:293:19
```

**Comment by Glutexo** [comment]
Confirmed: Using a local copy of this gem checked-out to 55a4ec9 (a commit right before merging this PR) solved this issue for me. This should be probably addressed. Can I help somehow with debugging?

**Pull request merged**
